mill-mima/src/com/github/lolgab/mill/mima/MimaBase.scala

```diff
@@ -113,72 +113,82 @@ private[mima] trait MimaBase
     */
   def mimaCurrentArtifact: T[PathRef] = T { jar() }
 
-  def mimaReportBinaryIssues(): Command[Unit] = T.command {
-    def prettyDep(dep: Dep): String = {
-      s"${dep.dep.module.orgName}:${dep.dep.version}"
+  def mimaReportBinaryIssues(): Command[Unit] = {
+    // only used for some sanity checks
+    val scalaBinVersionTask = this match {
+      case m: ScalaModule => T.task { scalaBinaryVersion(m.scalaVersion()) }
+      case _              =>
+        // FIXME: ask Mill, but `mill.main.BuildInfo` isn't available in older versions
+        T.task("2.13")
```

Comment:
```suggestion
      case _              => T.task("3")
```
I would set as "3" so it will continue to work for at least two decades. We don't even need to change it to `mill.main.BuildInfo`

Reply:
Actually, it can be `null`. Let me push a commit on top

Reply:
We could also remove the whole sanity check or make it optional by using an `Option[String]` for the Scala version.

Reply:
I used String or `null` since `Option` can't be used directly with the Java API, and I didn't want to use java's `Optional`.
I pushed a commit on top